src/Forms/FieldList.php

```diff
@@ -59,28 +59,86 @@ public function __clone()
     }
 
     /**
+     * Return an array of all fields, including deeply nested fields.
+     *
+     * @var FieldList
+     */
+    public function nestedFields()
```

Comment:
I  think this method name is ambiguous, wouldn't something like "flatten" be better?